README.md

```diff
@@ -1478,6 +1500,45 @@ When everything is working as expected locally using `mp-test`, we can run
   Using the collection error message along with the `adapter.log` can help trace the cause of the issue.
 </details>
 
+<details>
+  <summary><h3>Unable to build pak file after running mp-build?</h3></summary>
+
+  In most cases, this error indicates issues with building the container image. The most probable causes are:
+
+  1. Syntax error in the Dockerfile:
+
+  ```
+  mp-build
+  Building adapter [Finished]
+  Unable to build pak file
+  ERROR: Unable to build Docker file at /Users/user/code/management-packs/test:
+   {'message': 'dockerfile parse error line 6: unknown instruction: SYNTAX'}
```

Comment:
This message seems to indicate case 2 (unknown instruction). Are these distinct enough to have separate cases? They have the same solution below.